### ISSUE TITLE
How to Resume and Download Only Missing Samples Without Re-downloading the Entire Dataset

### ISSUE THREAD

**Issue by Ethan16162** [EvolvingLMMs-Lab/lmms-eval#853]
When evaluating **VideoMME**, since the dataset is quite large and the network is unstable, some samples failed to download. How can I **resume and download only the missing samples** to avoid deleting the already downloaded data and re-downloading the entire dataset?

`Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32) 
Model Responding:   1%|▍                                           | 6/676 [00:42<1:17:26,  6.94s/it]video path:/data/huggingface/videomme/data/YPaLGP_xr-w.mp4 does not exist, please check
video path:/data/huggingface/videomme/data/YPaLGP_xr-w.mp4 does not exist, please check`

**Comment by kcz358** [comment]
You can download the video dataset first (retry or resume) from the hub. Then extract them to the cache_dir. Then in evaluation, the model will skip the extraction part

**Comment by Luodian** [comment]
Looks like this was resolved. Closing - reopen if needed!
